wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/WorkspaceRuntimes.java

```diff
@@ -993,7 +1005,13 @@ public void run() {
         if (e instanceof InternalInfrastructureException) {
           LOG.error(e.getLocalizedMessage(), e);
         }
-        publishWorkspaceStatusEvent(workspaceId, STOPPED, STARTING, e.getMessage());
+        publishWorkspaceStatusEvent(
+            workspaceId,
+            STOPPED,
+            STARTING,
+            e.getMessage(),
+            isWorkspaceStartInterrupted,
+            Collections.emptyMap());
```

Comment:
I think we don't need to pass `emptyMap()` explicitly as we have overloaded method that is doing that for us